--- unifi.init.orig	2018-03-20 09:54:58.000000000 +0200
+++ unifi.init	2018-03-20 09:56:39.000000000 +0200
@@ -198,8 +198,8 @@
  -cp /usr/share/java/commons-daemon.jar:${BASEDIR}/lib/ace.jar \
  -pidfile ${PIDFILE} \
  -procname ${NAME} \
- -outfile SYSLOG \
- -errfile SYSLOG \
+ -outfile ${LOGDIR}/unifi.out.log \
+ -errfile ${LOGDIR}/unifi.err.log \
  -umask ${UMASK} \
  -user ${UNIFI_USER} \
  ${JSVC_EXTRA_OPTS} \
